debian/changelog

```diff
@@ -1,3 +1,11 @@
+tactile (4.3.0-0) unstable; urgency=low
+
+  * Change default text formatting to normal
+  * Fix cta text in bold
+  * add documentation for linux
```

Comment:
Only put features here so this isn't necessary.

```suggestion
```

Reply:
ok